types/auth0/index.d.ts

```diff
@@ -1312,6 +1312,27 @@ export interface VerifyEmail {
     } | undefined;
 }
 
+export interface GetDeviceCredentialsParams {
+    user_id: string;
```

Comment:
is any of those params required? like `user_id`?

Reply:
The auth0 module will accept anything, however the `user_id` field is required by the `/api/v2/device-credentials` endpoint. The endpoint's documentation says it 'Retrieves device credentials for a given user_id'. Using `getDeviceCredentials` without `user_id` results in this response: _Query validation error: 'Object didn't pass validation for format user-id: ' on property user_id (user_id of the devices to retrieve)._